### DISCUSSION TITLE
GitVersion appears to hang indefinitely in Windows Server 2019

### DISCUSSION THREAD

**Discussion by tehkyle** [GitTools/GitVersion#4106] [Q&A]
We have a azure devops yaml build pipeline that is running gitversion/setup@1 with 5.x and gitversion/execute@1 with updateAssemblyInfo: true.
Our execute task seems to hang indefinitely without log information in devops, going on over 2 hours before the pipeline times out. If we update to a windows server 2022 machine it seems to run ok. Running locally in command line on 2019 it writes the found tags then hangs. We are unable to use 2022 in our actual process, so want to know if there is any extra information we should be looking at for why this is happening.

Thank you

**Comment by HHobeck** [comment]
The version 5.x is not supported anymore. Please use the latest version and try again.

I'm going to close this discussion. Thank you for your understanding.
